.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,9 +6,13 @@
 ### Merge instructions
 <!-- If you want us to merge this PR as soon as we've reviewed, check the box below. If you're waiting for a release or there are other considerations that you want us to be aware of, list them below. -->
 
-- [ ] Please merge after reviewing
+Merge queue is enabled in this repo. If you'd like your PR to be merged immediately after it has receieved the required reviews, create the PR and then post the following comment:
```

Comment:
```suggestion
Merge queue is enabled in this repo. If you'd like your PR to be merged immediately after it has received the required reviews, create the PR and then add the following comment:
```

Reply:
I don't have a strong opinion on "post" -> "add" but it sounds a bit more natural to me.